utils_cv/detection/plot.py

```diff
@@ -0,0 +1,279 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""
+Helper module for visualizations
+"""
+from typing import List, Union, Tuple, Callable, Any
+from pathlib import Path
+
+import cv2
+import numpy as np
+import matplotlib.pyplot as plt
+from PIL.JpegImagePlugin import JpegImageFile
+
+from .references.coco_eval import CocoEvaluator
+
+
+class BoundingBoxParams:
+    """ Simple class to contain bounding box params. """
+
+    def __init__(
+        self,
+        rect_th: int = 2,
+        rect_color: Tuple[int, int, int] = (255, 0, 0),
+        text_size: float = 1,
+        text_th: int = 2,
+        text_color: Tuple[int, int, int] = (255, 255, 255),
+    ):
+        self.rect_th, self.rect_color, self.text_size, self.text_th, self.text_color = (
+            rect_th,
+            rect_color,
+            text_size,
+            text_th,
+            text_color,
+        )
+
+
+def plot_boxes(
+    im: JpegImageFile,
```

Comment:
Never seen this before "JpegImageFile". Maybe this is the right thing to do, maybe it's over-kill: im is just a numpy array albeit of (w x h x c) with c = {1,3}

Reply:
yeah, its a cv2 thing, if we switch to pillow we just use the np array then.